src/EnergyPlus/UnitarySystem.cc

```diff
@@ -15030,13 +15044,14 @@ namespace UnitarySystems {
                         HeatingCoils::SimulateHeatingCoilComponents(state,
                                                                     CompName,
                                                                     FirstHVACIteration,
-                                                                    DataLoopNode::SensedLoadFlagValue,
+                                                                    PartLoadFrac,
```

Comment:
More supp heat setpoint control fixes in `controlSuppHeatSystemToSP`.

Reply:
If not SensedLoadFlagValue then this argument is the load. So should use capacity * PLR ? Are any of the diffs a result of this change ??

Reply:
> If not SensedLoadFlagValue then this argument is the load. So should use capacity * PLR ? Are any of the diffs a result of this change ??

This argument is `QCoilReq` in `SimulateHeatingComponents`. Technically this should be `PartLoadFrac * this->m_DesignSuppHeatingCapacity` but `PartLoadFrac = 0.0 here to get the no-load result, so just being lazy here. Updated the end-line comment to help clarify, maybe.

Reply:
I see. I didn't notice this was a PLR=0 call.